docker/Dockerfile

```diff
@@ -1,6 +1,7 @@
+#docker build -t  openmmlab/lmdeploy-builder:cuda12.2 . -f docker/Dockerfile
 ARG CUDA_VERSION=cu12
 
-FROM nvidia/cuda:12.4.1-devel-ubuntu22.04 AS cu12
+FROM nvidia/cuda:12.2.0-devel-ubuntu22.04 AS cu12
```

Comment:
`12.4` is too high for your case?

Reply:
yes，if image's cuda version higher than host machine's cuda version,it would't be compatibility

Reply:
Hi, personally, one should build his customized docker image and change the dockerfile when needs.

Reply:
> Hi, personally, one should build his customized docker image and change the dockerfile when needs. I do not think it is a ideal solution to change to cuda12.2

i wil change it back to 12.4.1

Reply:
Please rollback to `nvidia/cuda:12.4.1-devel-ubuntu22.04 AS cu12`
If incompatibiltiy happens, users can choose `openmmlab/lmdeploy:latest-cu11`